Nos/Views/RepliesView.swift

```diff
@@ -163,6 +165,14 @@ struct RepliesView: View {
                 }
             }
         }
+        .task {
+            await computeDirectReplies()
+        }
+        .onChange(of: replies.count) { _ in
```

Comment:
There is a potential bug here where the contents of `replies` could change but the count stays the same... I don't see an easy way to work around this. A quick glance at SwiftData's new `@Query` seems like it might be a solution but that's too heavy a lift to do right now. Just something to keep in mind for the future I guess.

Reply:
Yes, I thought about that. In my prior experiencie, listening to collection updates is really expensive in SwiftUI so I chose to listen to the count instead because it's a good heuristic in this case as replies are not likely to change